aspnetcore/fundamentals/localization.md

```diff
@@ -343,4 +343,5 @@ Terms:
 * [Globalizing and localizing .NET applications](/dotnet/standard/globalization-localization/index)
 * [Resources in .resx Files](/dotnet/framework/resources/working-with-resx-files-programmatically)
 * [Microsoft Multilingual App Toolkit](https://marketplace.visualstudio.com/items?itemName=MultilingualAppToolkit.MultilingualAppToolkit-18308)
+* [Localization & Generics](https://github.com/hishamco/hishambinateya.com/blob/master/Posts/localization-and-generics.md)
 * [Generic Class and Localization](https://github.com/aspnet/AspNetCore.Docs/issues/8244)
```

Comment:
I'm not sure if we need this one,

Reply:
Delete https://github.com/aspnet/AspNetCore.Docs/issues/8244 ?

Reply:
Yep, because the article is answer for that issue

Reply:
I thought so but the blog is highlighted in GitHub.